src/view/components/wrappedField.jsx

```diff
@@ -81,6 +89,9 @@ class DecoratedInput extends React.Component {
       fieldComponentsProps.inputValue = input.value;
     }
 
+    const { onBlur } = fieldComponentsProps;
```

Comment:
It seems there was an issue inside @adobe/react-spectrum@3.7.0 package. The blur event was not triggered. When I upgraded to @adobe/react-spectrum@3.13.0, the blur event was triggered but it was generating an error. The blur event was triggered with a FocusEvent. Redux form doesn't work well when it receives a FocusEvent. When blur was occurring on a Picker component, the picker component value was changed to ''. By calling the onBlur event with the picker value, this works as expected now.

Reply:
Leave a smaller comment about this inside this file at these lines so we know why we're doing this.